pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.lfenergy.compas</groupId>
+  <artifactId>compas-scl-xsd</artifactId>
+  <version>0.1-SNAPSHOT</version>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.googlecode.maven-download-plugin</groupId>
+        <artifactId>download-maven-plugin</artifactId>
+        <version>1.6.0</version>
+        <executions>
+          <execution>
+            <id>get-1.7-xsd</id>
+            <phase>initialize</phase>
+            <goals>
+              <goal>wget</goal>
+            </goals>
+            <configuration>
+              <uri>https://assets.iec.ch/public/tc57/IEC_61850-6.2003.SCL.1.7.full.zip</uri>
```

Comment:
Does this mean that we need to change this uri every time a new version is being published by IEC?
If so, maybe this is not the best idea because it still requires some manual actions :)
And we want it to be automatic!